parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -385,6 +385,7 @@ def interchange_communicator(self, pair_setup: threading.Event):
                 meta_b = pickle.dumps({'type': 'result'})
                 ix_sock.send_multipart([meta_b, results_sock.recv()])
                 logger.debug("Result sent to interchange")
+
```

Comment:
was this blank intentional?

Reply:
Yeah; I should have added in a previous PR, to make it congruent with the other branches.  Trying to sneak it in!